README.md

```diff
@@ -15,3 +16,87 @@ contents of this repo in parts or in whole according to the BSD0 license:**
 
 This is an initial example of how to setup a marketplace where digital tokens
 can be issued and exchanged.
+
+# Requirements
+To run the marketplace locally or to build to be deployed on DAML, the following requirements are necessary:
```

Comment:
deployed on DAML, or on DABL?

Reply:
we should probably also add a small section for deploying + running on DABL: 

1) On DABL: Create a new project/ledger, add all the parties for the system
2) In repo: run `make package`
3) Upload artifacts from `target` directory to ledger
4) Deploy bots with respective parties
5) (optionally) run the bootstrap script by downloading the participants.json and doing all that stuff (forgetting the details atm, its been a while!)

Reply:
I agree with doing a DABL section - I'm planning to add it in a separate PR since it's a bit more involved, we have a partner that wants to develop locally ASAP so I'd rather get this out first.